src/sonic-config-engine/sonic_device_util.py

```diff
@@ -29,6 +31,39 @@ def get_machine_info():
             machine_vars[tokens[0]] = tokens[1].strip()
     return machine_vars
 
+def get_num_npus():
+   platform = get_platform_info(get_machine_info)
+   num_npus = 1
+   asic_conf_file_path = os.path.join(SONIC_DEVICE_PATH, platform, 'asic.conf')
+   if not os.path.isFile(asic_conf_file_path):
+	return 1
+   with open(asic_conf_file_path) as asic_conf_file:
```

Comment:
it looks like you are mixing tab and spaces, can you use spaces.

Reply:
Fixed in the latest commit